src/main/java/me/qoomon/maven/gitversioning/GitVersioningModelProcessor.java

```diff
@@ -135,7 +135,15 @@ public Model processModel(Model projectModel, Map<String, ?> options) throws IOE
                     return projectModel;
                 }
 
-                if (parseBoolean(getCommandOption(OPTION_NAME_DISABLE))) {
+                if (FALSE.equals(parseBoolean(getCommandOption(OPTION_NAME_DISABLE)))) {
+                    logger.info("non-skip - versioning is enabled");
```

Comment:
I think it's not necessary to log no-skip in case of normal behaviour of the extension.